tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -296,7 +296,7 @@ def add_dim3(kernel_string, cuda_kernel):
             if count < 2:
                 arg_locs[count]['start'] = ind + 1
 
-    first_arg_raw = kernel_string[arg_locs[0]['start']:arg_locs[0]['end'] + 1]
+    first_arg_raw = kernel_string[arg_locs[0]['start']:arg_locs[0]['end']]
```

Comment:
What's going on here? I thought we already fixed this in master?

Reply:
@Jorghi12 didn't think so in https://github.com/ROCmSoftwarePlatform/pytorch/pull/172

If you want to take this one out, I can do that. I'd close and make a new one just for the skip and hgemm